marxs/source/labSource.py

```diff
@@ -7,38 +7,29 @@
 
 
 class FarLabConstantPointSource(Source, FlatOpticalElement):
-    '''Simple in-lab source used with aperture
+    '''Simple in-lab point source used with aperture
 
     - assumes point source is far from a rectangular aperture, and only the photons that pass through are tracked
     - photon start positions uniformly distributed within rectangular aperture (reasonable approximation if source is far)
     - photon direction determined by location of source, and selected photon starting position
-    - aperture is parallel to y-z plane
-    - polarization angle is assumed relative to positive y axis unless direction is exactly along y axis,
-      in which case polarization is relative to positive x axis, ALL GLOBAL AXES
-    - TODO: figure out how to provide energy distribution
+    - aperture is in its local y-z plane
 
     Parameters
-    ----------------
-    sourcePos: 3 element list
+    ----------
+    sourcePos: 3 element list of floats
         3D coordinates of photon source (not aperture)
-    polarization: UNKNOWN
-        TODO: determine representation of polarization (needs magnitude and orientation, anything else?)
-    rate: float
-        photons generated per second
-    energy: UNKNOWN
-        TODO: determine representation of energy distribution
-    **kwargs: 'position', 'orientation', 'zoom'
-        4x4 pos4d matrix for transformations in homogeneous coords from local coords to global coords
+    **kwargs: 'position', 'orientation', 'zoom'; 'flux', 'energy', 'polarization'
+        first three - for calculating aperture 4x4 pos4d matrix for transformations in homogeneous coords from local to global
+        second three - choose photon distribution in time, energy, and polarization that source emits (see Source class)
```

Comment:
You can actually write
```
see `Source`
```
and sphinx will figure out that this is a Reference to something called "Source" and then set the appropriate link in html.
But, you don't have to Purely optional magic :-)